docs/README.md

```diff
@@ -150,14 +152,43 @@ XX is a number between 02-99. Currently, 00_revision_history.md
 contains the documentation revision history, and 01_overview.md is the
 overview of the compiler goals and architecture.
 
+## Quick guide for developing the Doxygen documentation
```

Comment:
```suggestion
## Building the Doxygen documentation
```

Also the C++ comments guide should be the most prominent. Doxygen should have less priority.